packages/components/src/Tooltip/Tooltip.tsx

```diff
@@ -68,6 +71,25 @@ export default function Tooltip({
   className,
   ...rest
 }: TooltipProps) {
+  const hideOnEsc: Plugin = {
+    name: "hideOnEsc",
+    defaultValue: true,
+    fn: ({ hide }) => {
+      function onKeyDown(event: KeyboardEvent) {
+        if (event.code === "Escape") {
```

Comment:
`event.keyCode` is deprecated, went with `event.code`

Reply:
I thiiink `event.key` is [slightly more recommended](https://developers.google.com/web/updates/2016/04/keyboardevent-keys-codes). seeing docs around that "code" is the physical key being pressed while "key" is the meaning of the key (in case someone applied a separate keyboard layout or something)

Reply:
Changed to `key`, thanks